dot/parachain/collator-protocol/message_test.go

```diff
@@ -172,3 +178,222 @@ func TestDecodeCollationHandshake(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, testHandshake, msg)
 }
+
+func TestHandleCollationMessageCommon(t *testing.T) {
```

Comment:
is there a happy path to test here?

Reply:
these are common errors that could happen. Happy path would change as per the message. There is a success case in TestHandleCollationMessageDeclare which is the happy path for that message.